packages/lab/src/components/Flow/stories/CustomDrop/LineChart.tsx

```diff
@@ -1,14 +1,14 @@
 import { css } from "@emotion/css";
 import { HvFlowNode } from "@hitachivantara/uikit-react-lab";
 import { HvLineChart } from "@hitachivantara/uikit-react-viz";
-import { NodeProps, useStore } from "reactflow";
+import { NodeProps, useEdges, useNodes } from "reactflow";
 
 import { data } from "./data";
 
 export const LineChart = (props: NodeProps) => {
   const { id } = props;
-  const nodes = useStore((state) => state.getNodes());
-  const edges = useStore((state) => state.edges);
+  const nodes = useNodes<any>();
```

Comment:
I think we should try to remove this `any`, but i couldn't find a quick replacement

Reply:
Yes, you are right. I'll take a look at this 🙏